addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -16,12 +16,22 @@ const TableOfContent = publicWidget.Widget.extend({
         this.$scrollingElement = this.$target.closest(".s_table_of_content").closestScrollable();
         this.previousPosition = -1;
         this._updateTableOfContentNavbarPosition();
-        extraMenuUpdateCallbacks.push(this._updateTableOfContentNavbarPosition.bind(this));
+
+        //  bind _updateTableOfContentNavbarPosition.
+        this.boundUpdateNavbar = this._updateTableOfContentNavbarPosition.bind(this);
+        extraMenuUpdateCallbacks.push(this.boundUpdateNavbar);
     },
     /**
      * @override
      */
     destroy() {
+        //  remove the bound function from the extraMenuUpdateCallbacks array so that
+        //  `_updateTableOfContentNavbarPosition` won't gets called after public widget
+        //  destroyed.
+        const indexOfCallback = extraMenuUpdateCallbacks.indexOf(this.boundUpdateNavbar);
+        if (indexOfCallback !== -1) {
```

Comment:
How can we end up with `extraMenuUpdateCallbacks` that does not contain `this.boundUpdateNavbar` ? :thinking: